### ISSUE TITLE
Allow installing database G05 and W08 for solver.

### ISSUE THREAD

**Issue by han-k59** [artyom-beilis/OpenLiveStacker#23]
Solving with the 50 mm lens is easy and reliable. ASI1600 was uncooled. A one second exposure image is easy to solve. See this example 

https://nova.astrometry.net/user_images/7881879

But I have to install the G05 database manually. Would be better to allow installing G05 and W08 for wide field-field solving

**Comment by artyom-beilis** [comment]
> But I have to install the G05 database manually. 

There is no G05 zip file at https://sourceforge.net/projects/astap-program/files/star_databases/

w08 I can add easily.

**Comment by han-k59** [comment]
I have added it now:

https://sourceforge.net/projects/astap-program/files/star_databases/g05_star_database.zip/download

**Comment by artyom-beilis** [comment]
I tried ASTAP with g05 and for some reason it complains that no DB found.

w08 was used with wide field of fiew but g05 ignored even if I try to run with FOV=1.0

**Comment by han-k59** [comment]
The program will only look for/select the G05 if the FOV is larger then 10 degrees.
Same applies for the W08. This one will only be selected if FOV is larger then 20 degrees.


So the W08 should not work unless you have a real wide field image. I will look into the code.

**Comment by artyom-beilis** [comment]
Quoting readme:

> G05 star database for 20>FOV>0.6 degrees

Should it be 20>FOV>10 degrees ?

Also I try to run I get an error

```
$ astap_cli -f raw_stretched.png -fov 15.0 -ra 7.614167 -spd 155.610000 -r 35.000000 -d db/
Error, no star database found at db// ! Download and install a star database.
```

Only when I unzip d05 it stars using g05

```
Using star database D05
Large FOV, use G05 database! 
```

Is it expected behaviour? If I need to use G05, I need to have D05 installed as well?

**Comment by han-k59** [comment]
I noted a typo error in the code. The g05 is only selected if no other database is available. If the FOV is larger then 10 degrees then it is looking for capital written G05. But it is in lower case so it will not find it in Linux and Android. In Windows the file name with or without capital letter doesn't matter. I will fix this today in astap_cli.

**Comment by han-k59** [comment]
It will work for this range: 
G05 star database for 20>FOV>0.6 degrees (capability)
But it uses larger tiles so is slower. So it is better to recommend it for this range
G05 star database for 20>FOV>1 degrees (recommendation for hard disk)
or maybe for a slow SD card this
G05 star database for 20>FOV>10 degrees (recommendation for SD card?)

Let me speed test it on a Raspberry Pi to see how the G05 performs  compared with the D05. The only difference is the database tile size.

> Is it expected behaviour? If I need to use G05, I need to have D05 installed as well?

No. Is not required
An other typo in my code. It is checking for g05_0101.1476 but it should look for g05_0101.290 !!! It will only work if the FOV>10.

I will correct it.....

**Comment by han-k59** [comment]
I have corrected and uploaded the Android versions. The FOV where the G05 is better then the D05 is depending on speed of disk assessment. For the moment keep the selection as  follows:

     D50      (10 > fov > 0.2 degrees)
     D20      (10 > fov > 0.3 degrees) 
     D05      (10 > fov > 0.6 degrees) 
     G05      (20 > fov > 3.0 degrees) 
     W08      (       fov > 20 degrees)

**Comment by artyom-beilis** [comment]
Fixed in 254d01718e32b91d1f41301a7377ac82c7f62e35
